js/login.js

```diff
@@ -0,0 +1,68 @@
+import { browserHistory } from 'react-router';
+import Service from './service.js';
+import UserData from './user-data.js';
+
+export default {
+  clearLoggedInParam(locationObj) {
+    let query = locationObj.query;
+
+    delete query.loggedin;
+
+    browserHistory.replace({
+      pathname: locationObj.pathname,
+      query: query
+    });
+  },
+  verifyLoggedInStatus(currentLocation, callback) {
+    let loggedInParam = currentLocation.query.loggedin;
+
+    if (loggedInParam) {
+      // After the login process user will be redirected back to Pulse.
+      // There will be a query argument "loggedin" presented in the URL
+      // which is something we don't want it to stay at all times.
+      this.clearLoggedInParam(currentLocation);
+    }
+
+    // verify user's logged in status with Pulse API
+    Service.userstatus()
+      .then(response => {
+        let type = `unknown`;
```

Comment:
For v2, we don't need these checks. Someone is either staff (or at least someone we trust enough to give a moz ldap account), or they're an anonymous user right now.